db/migrate/20200514162708_add_apple_authentication_fields.rb

```diff
@@ -0,0 +1,6 @@
+class AddAppleAuthenticationFields < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :apple_created_at, :datetime
```

Comment:
Not for this PR, but in the future we might consider breaking out auth columns to a separate table as we add additional auth providers!